Makefile

```diff
@@ -20,5 +20,5 @@ lint:
 publish:
 	./scripts/publish.sh ${GITHUB_RUN_NUMBER}
 
-unit-test:
+test:
```

Comment:
we need to update the `main.yml` to reflect this change, then the coverage report should go through again 🤞